Edits to DOCS-1076 - Documenting PGD 5.6.1 #6210 (#6298)
EnterpriseDB/docs: product_docs/docs/pgd/5.6/ddl/ddl-command-handling.mdx

-   `ADD COLUMN ... DEFAULT (non-immutable expression)` &mdash; This is not allowed because
    it currently results in different data on different nodes. See
    [Adding a column](ddl-workarounds/#adding-a-column) for a suggested workaround.

>     This can be overridden using `bdr.permit_unsafe_commands` if user is sure the command is

Author: Is there some reason this is in third person and passive voice? Suggest this revision:

```suggestion
    You can override this behavior using `bdr.permit_unsafe_commands` if you're sure the command is
```

    safe.
-   `ALTER COLUMN ... SET STORAGE external` &mdash; Is rejected if the column is
    one of the columns of the replica identity for the table.

>     This can be overridden using `bdr.permit_unsafe_commands` if user is sure the command is

Author: Same query. Suggest:
```suggestion
    You can override this behavior using `bdr.permit_unsafe_commands` if you're sure the command is
```
